ratis-shell/pom.xml

```diff
@@ -24,9 +24,45 @@
   <name>Apache Ratis Shell</name>
 
   <properties>
+    <guava.version>29.0-jre</guava.version>
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>${guava.version}</version>
+      <exclusions>
+        <exclusion>
+          <artifactId>jsr305</artifactId>
+          <groupId>com.google.code.findbugs</groupId>
+        </exclusion>
+      </exclusions>
+    </dependency>
```

Comment:
guava is already shaded in ratis.  So we don't need to add it.

Reply:
Yeah, remove it